cognite/neat/_rules/analysis/_base.py

```diff
@@ -490,40 +490,27 @@ def _dms_di_graph(self, format: Literal["data-model", "implements"] = "data-mode
         # Views with properties or used as ValueType
         # If a view is not used in properties or as ValueType, it is not added to the graph
         # as we typically do not have the properties for it.
-        used_views = {prop_.view for prop_ in rules.properties} | {
```

Comment:
Remember to remove the comment above

Reply:
What it the motivation for including all views, also the ones without properties?